samcli/lib/sync/infra_sync_executor.py

```diff
@@ -179,8 +196,18 @@ def execute_infra_sync(self, first_sync: bool = False) -> InfraSyncResult:
             #     )
             pass
 
+        # Will be added with the sync infra skip is ready for release
+        # if days_since_last_infra_sync > AUTO_INFRA_SYNC_DAYS:
```

Comment:
I wonder if we can simplify this double if blocks over same condition. Not blocking now but something for GA PR.

Reply:
Yeah that makes sense. This if loop can be switched to an elif condition with the above if condition.